### ISSUE TITLE
JWT Algorithm Confusion Security vulnerability with fast-jwt < 3.3.2

### ISSUE THREAD

**Issue by markrzen** [nearform/fastify-jwt-jwks#12]
REL: https://github.com/advisories/GHSA-c2ff-88x2-x9pg

**Comment by markrzen** [comment]
@simoneb I noticed you fixed fast-jwt, any chance you are already updating this repository as well?

**Comment by simoneb** [comment]
Good point, I'm not quite sure why dependabot hasn't bumped fast-jwt in fastify/jwt.

**Comment by markrzen** [comment]
Happy to put in a quick PR, just didn't want stomp on your toes.

As a side note: I think `fastify-auth0-verify` is in an odd state, given that it's own fast-jwt has been upgraded, but it's dependency on this library, pulls in the older fast-jwt.

**Comment by simoneb** [comment]
Please do, I was largely relying on dependabot doing its job, but for whatever reason this is not happening. The dependencies are using loose semver specifiers, which would normally not receive bump PRs, but in case of advisories I was definitely expecting dependabot to kick in. Feel free to send PRs, I would appreciate it.

As for fastify-auth0-verify, I believe the dependency on @fastify/jwt is redundant

**Comment by markrzen** [comment]
@simoneb Upgraded both, tests past. Best I can tell the fastify/jwt major version upgrade doesn't impact this lib, as the key prop isn't exposed.

**Comment by markrzen** [comment]
@simoneb Any ETA on when this can be published?
